src/routes/labs-route.js

```diff
@@ -0,0 +1,70 @@
+const config = require('config');
+const multer = require('multer');
+const GridFsStorage = require('multer-gridfs-storage');
+const path = require('path');
+const crypto = require('crypto');
+const express = require('express');
+
+const storage = new GridFsStorage({
+  url: config.mongo,
+  connectionOpts: { useNewUrlParser: true },
+  file: (req, file) => {
+    return new Promise((resolve, reject) => {
+      crypto.randomBytes(16, (err, buf) => {
```

Comment:
Crypto library should be used with care cause it is using the 'thread pools' in node.js this can be a performance leak. I suggest for this case to use the timestamp instead of crypto.randomBytes